parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -122,7 +122,7 @@ configureCustomCaCertificate() {
         declare varname=CUSTOM_CA_CERT_${i} 
         echo "${!varname}" > /opt/certs/00000000000000cert${i}.crt
     done
-    systemctl restart update_certs.service || exit $ERR_UPDATE_CA_CERTS
+    systemctl restart update_certs_provisioning.service || exit $ERR_UPDATE_CA_CERTS
```

Comment:
```suggestion
    # this block until the service is considered active!
    systemctl restart update_certs || exit $ERR_UPDATE_CA_CERTS
    systemctl restart containerd
```

now make update_certs.service `type=Notify`, and add `systemd-notify --ready` to the end of update_certs.sh

now `systemctl restart update_certs` blocks until `systemd-notify --ready` was called, and you don't need the extra unit